src/Microsoft.DotNet.Interactive.Formatting.Tests/PlainTextFormatterTests.cs

```diff
@@ -15,22 +15,6 @@ namespace Microsoft.DotNet.Interactive.Formatting.Tests
 {
     public class PlainTextFormatterTests : FormatterTestBase
     {
-        [Fact]
```

Comment:
These tests are removed because the returned formatter is now often a composition of possible formatters (using AnonymousTypeFormatter) and its not useful to check the runtime type of the formatter on return.